src/components/loader/loader.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
```

Comment:
comment: I don't think we need to import this in either file, if we do it should be done in the jest config (I think it is already but could be wrong)